__tests__/Result_test.re

```diff
@@ -1,13 +1,10 @@
 open Jest;
-open Expect;
-open Rebase;
```

Comment:
Why did this need to be moved locally? The idea of a standard library replacement in particular is that it should override standard library modules, and if that's not possible for some reason it ought to have consequences for its design.

Reply:
>  idea of a standard library replacement in particular is that it should override standard library modules, and if that's not possible for some reason it ought to have consequences for its design.

I was trying to figure out if there were conflicting functions. Thanks for the guidance on the idea behind it.

Reply:
No problem :)